src/components/SelectionList/Search/UserInfoCell.tsx

```diff
@@ -17,6 +17,10 @@ function UserInfoCell({participant, displayName}: UserInfoCellProps) {
     const {isLargeScreenWidth} = useResponsiveLayout();
     const avatarURL = participant?.avatar;
 
+    if (displayName === CONST.REPORT.OWNER_EMAIL_FAKE || displayName === '') {
```

Comment:
I know that the `transactionItem.managerID === 0` check, which if true returns `emptyPersonalDetails` would always return the `displayName` as empty string, which would make the next paragraph non-valid but I thought I should mention it for the case (not sure it will ever exist) where `transactionItem.managerID !== 0` but the displayName is `__fake__`.

I noticed in the Slack [conversation](https://expensify.slack.com/archives/C049HHMV9SM/p1725454490800639) screenshot (see below) that the `__fake__` displayName was lowercase and here we're checking for equality between `displayName` (which can be lowercase) and `CONST.REPORT.OWNER_EMAIL_FAKE` which is `__FAKE__` (uppercase) - which might cause an edge case where this logic won't return early because both checks will fail.

<img width="380" alt="Screenshot 2024-09-13 at 17 25 21" src="https://github.com/user-attachments/assets/985e37a9-9b27-4151-a57c-918b26aa0821">

##

As a fix I'd recommend either uppercasing `displayName` for this check, or lowercasing both.

@luacmartins Wdyt ?

Reply:
Ah nice catch. Yea, we should make the comparison case insensitive

Reply:
makes sense and I can update, but then Im curious why we get the lowercase `__fake__` in the first place? Is it backend that produces this value? In the `App` I get 0 results when searching for `__fake__`.